bitnami/solr/README.md

```diff
@@ -179,6 +179,7 @@ The following tables lists the configurable parameters of the solr chart and the
 | `ingress.pathType`               | Path type for the ingress resource                       | `ImplementationSpecific`       |
 | `ingress.path`                   | Default path for the ingress resource                    | `/`                            |
 | `ingress.tls`                    | Create TLS Secret                                        | `false`                        |
+| `ingress.existingSecretName`     | The name of an existing Secret to use for the ingress object | `nil`                          |
```

Comment:
In RabbitMQ we are naming it `ingress.existingSecret`. For consistency purposes, I'd recommend to standarize into that. Could you change it?

Reply:
updated